TwinCAT.gitignore

```diff
@@ -0,0 +1,15 @@
+# gitignore template for TwinCAT
+# website: https://www.beckhoff.com/twincat/
+#
+# Recommended: VisualStudio.gitignore
+
+# TwinCAT files
+*.tpy
+*.tclrs
+*.compiled-library
+*.compileinfo
+*.tmc
```

Comment:
Commiting TMC is up to the user
```suggestion
#*.tmc
```

Reply:
No it's not. Since 4018 Beckhoff specifically says tmc should be ignored (other for the exception of C++, which is also explicitly mentioned).